### DISCUSSION TITLE
Hello I have a problem

### DISCUSSION THREAD

**Discussion by Scrapper142** [community/community#121803] [Actions]
### Select Topic Area

Show & Tell

### Body

How do you think I can solve this problem?

![Ekran görüntüsü 2024-05-01 132943](https://github.com/community/community/assets/71039310/406d9a9a-71e9-456d-b174-40adf38b114c)

**Comment by gurjeetsinghvirdee** [comment]
Hey @Scrapper142 

Raise a ticket as it mentioned here https://support.github.com/, they will help you for sure

**Comment by notAnii** [comment]
You could try contacting GitHub support, sometimes the system can falsely flag one of your repos
